.github/workflows/release.yml

```diff
@@ -33,7 +38,43 @@ jobs:
 
       - run: make -C .ci dist
 
-      - run: make -C .ci push-docker
+      - name: Attest Lambda layer zip
+        uses: github-early-access/generate-build-provenance@main
+        with:
+          subject-path: "${{ github.workspace }}/build/aws/elastic-apm-node-lambda-layer-*.zip"
```

Comment:
Q: Do we really want to (a) publish attestations (this goes on a public immutable ledger of some sort, IIUC), and (b) docker build and push for **every commit to main**?  My concern is that this is very wasteful.

Reply:
I'm probably biased, but I prefer to run the release on the main branch with a `dry-run` or a subset of the jobs; this can detect any breaking changes in workflows.

Therefore, I enabled the provenance for those artefacts - I see it might be useful, and those artefacts will be available and with their attestations.

From the disc space, it should be ok, there is no limitation in our elastic docker registry